packages/UsageAnalysis/files/Test Track/Viewers/Heatmap.md

```diff
@@ -9,8 +9,22 @@
     1. Go to the grid.
     2. Right-click the Primary Series Name column’s header and select **Sort > Custom** from the context menu.
     3. Move the empty value to the first place. Click Apply, OK
-    4. Right-click the Primary Series Name column’s header and select **Sort > Ascending** (empty value should be in the first place)**, Descending.**
+    4. Right-click the Primary Series Name column’s header and select **Sort > Ascending** (empty value should be in the first place), **Descending**.
+7. Layouts saving & visualization:
+* Open new **SPGI** dataset. On the **Viewers tab**, click **Heatmap**.
+* In the **Property Pane**:
+  * increase 'Max Heatmap Columns' to 100.
+  * switch off 'Is heatmap' option.
+  * move scrollbar position
+* Save and re-apply layout.
+* Switch on 'Is heatmap' option in the **Property Pane**. 
+* **Expected Results**: 
+  * After applying the layout, the Heatmap visualization should be identical to the view before the 'Is heatmap' option was switched off. 
+  * The scrollbar position should be saved for layout
+  * There should be no visible differences in the Heatmap display.
+
 ---
 {
   "order": 7
+  "datasets": ["System:DemoFiles/SPGI.csv"]
```

Comment:
Link all datasets to the test case

Reply:
added  "datasets": ["System:DemoFiles/SPGI.csv","System:DemoFiles/SPGI-linked1.csv","System:DemoFiles/SPGI-linked2.csv"]